mdLinks.js

```diff
@@ -0,0 +1,245 @@
+	const dirWithRead = fs.readdirSync(files, 'utf-8');
+  return dirWithRead;
+}
+
+// Función para un archivo.
+// const isFile = (file) => {
+//   if (file === undefined) {
+//     console.log('Ingresa el archivo')
+//   } else {
+//     const extension = path.extname(file);
+//     if (extension === '.md') {
+//       const content = readFile(file);
+//       if (comands['validate'] === '--v' || comands['validate'] === '--validate') {
+//         return validateLinks(content)
+//       } else if (comands['stats'] === '--s' || comands['stats'] === '--stats') {
+//         validateLinks(content)
+//           .then((arrayLinks) => {
+//             const stats = statsStadistics(arrayLinks);
+//             return Promise.all(stats);
+//           }).then((res)=> {
+//             console.log(res)
+//           })
+//           .catch((err) => console.log(err));
+//       } else {
+//         return getLinks(content);
+//       }
+//     } else {
+//       console.log('Solo archivos .md');
+//       return false; 
+//     }; 
+// }  
+// }
+
+
+
+// Función para una carpeta.
+// const isDir = (files) => {
+//   const contentDir = filesDir(files);
+//   console.log(contentDir, 'gatitos');
+//   contentDir.forEach((file) => {
+//     if (file === undefined) {
+//       console.log('Ingresa el archivo')
+//     } else {
+//       const extension = path.extname(file);
+//       if (extension === '.md') {
+//         const content = readFileDir(files, file);
+//         if (comands['validate'] === '--v' || comands['validate'] === '--validate') {
+//           return validateLinks(content)
+//         } else if (comands['stats'] === '--s' || comands['stats'] === '--stats') {
+//           validateLinks(content)
+//             .then((arrayLinks) => {
+//               console.log(arrayLinks, 'perritos');
+//               const stats = statsStadistics(arrayLinks);
+//               return Promise.all(stats);
+//             }).then((res)=> {
+//               console.log(res, 'ay')
+//             })
+//             .catch((err) => console.log(err));
+//         }  else {
+//           return getLinks(content);
+//         }
+//       } else {
+//         console.log('Este no es un archivo .md');
+//         return false; 
+//       };     
+//     };
+//   }) 
+// }
+
+// const readArchive = (md) => {
+//   const content = readFile(md);
+//   const links = getLinks(content);
+//       if (comands['validate'] === '--v' || comands['validate'] === '--validate') {
+//         return validateLinks(links)
+//       } else if (comands['stats'] === '--s' || comands['stats'] === '--stats') {
+//         return statsStadistics(links)
+//       } else {
+//         return getLinks(content);
+//       }
+// }
+
+const mds = (filePath) => {
+  let md = [];
+  const extension = path.extname(filePath);
+  if (extension === '.md') {
+    let arr = md.push(filePath); 
+    console.log(arr, 'cerditos');   
+  } else if (searchDir(filePath)) {
```

Comment:
`else` o `else if is directorio`. Usa `isDirectory` para saber

Reply:
Perfecto! ya usado